### PR TITLE
fix(toolbar): assign label to icons

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

There were some accessibility issues related to the labels of icons and buttons in our toolbar. This PR should fix it. 

## Testing

CI should pass

## Docs

N/A

### CODE REVIEW

**Review by delucis — CHANGES_REQUESTED** [review]
We shouldn’t use `<title>` here as it introduces a duplicate tooltip for those using a mouse to hover:

toolbar icon with a “Menu” tooltip and also the browser’s native title tooltip reading “Menu” overlapping

I also did some screenreader testing and it appears these changes are not required — the buttons already include accessible labels in the accessibility tree:

Firefox accessibility tree visualization showing 4 buttons with labels of Menu, Inspect, Audit, and Settings

Instead, we should hide the SVGs using `aria-hidden="true"``.

File: packages/astro/src/runtime/client/dev-toolbar/toolbar.ts [hunk 1]
```diff
@@ -416,9 +416,9 @@ export class AstroDevToolbar extends HTMLElement {
 	}
 
 	getAppTemplate(app: DevToolbarApp) {
-		return `<button class="item" data-app-id="${app.id}">
+		return `<button class="item" data-app-id="${app.id}" aria-labelledby="app-name-${app.name}">
 				<div class="icon">${app.icon ? getAppIcon(app.icon) : '?'}<div class="notification"></div></div>
-				<span class="item-tooltip">${app.name}</span>
+				<span class="item-tooltip" id="app-name-${app.name}">${app.name}</span>
 			</button>`;
 	}
```
Comment (delucis) [inline packages/astro/src/runtime/client/dev-toolbar/toolbar.ts:423]: Could you explain what’s intended here? I’m not 100% sure where this code is used, but in theory `aria-labelledby` should be unnecessary here as a `<button>` is already automatically labelled by its text content.

**Review by delucis — COMMENTED** [review]
Thanks for the changes @ematipico! They look good. Just one question.

File: packages/astro/src/runtime/client/dev-toolbar/toolbar.ts (hunk 1 above)
Comment (ematipico) [reply packages/astro/src/runtime/client/dev-toolbar/toolbar.ts]: I don't recall. I removed it :)

**Review by delucis — APPROVED** [review]
Perfect. OK, should be good to go I think! Thanks for bearing with me on this one 🙌

**Pull request merged**